config/prod/cis-for-beanstalk-image-pipeline.yaml

```diff
@@ -0,0 +1,9 @@
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.9.6/templates/ImageBuilder/cis-for-eb-image-pipeline-template.yaml"
+stack_name: "cis-for-beanstalk-image-pipeline"
+stack_tags:
+  OwnerEmail: "it@sagebase.org"
+  CostCenter: "Platform Infrastructure / 990300"
+parameters:
+  ImageVersion: "1.0.0"
```

Comment:
i have general concerns in the dependent PR related to this parameter and other things.  I've decided to make comments there even though it has already been merged.

Reply:
> i have general concerns in the dependent PR related to this parameter and other things. I've decided to make comments there even though it has already been merged.

See https://github.com/Sage-Bionetworks/aws-infra/pull/427

Reply:
thanks